packages/server-wallet/src/channel-manager.ts

```diff
@@ -0,0 +1,92 @@
+import {CreateChannelParams} from '@statechannels/client-api-schema';
+import {backOff, IBackOffOptions} from 'exponential-backoff';
+
+import {MessageServiceInterface} from './message-service/types';
+import {getMessages} from './message-service/utils';
+import {WalletObjective} from './models/objective';
+import {Wallet} from './wallet';
+export const delay = async (ms = 10): Promise<void> =>
+  new Promise(resolve => setTimeout(resolve, ms));
+
+export class ChannelManager {
```

Comment:
I think we should discuss naming tomorrow in standup. Though I think it's fine to use `ChannelManager` here, we all want to call it `Wallet`. Tom suggested `Engine` or `WalletEngine` for what's currently in `wallet.ts`, which I think works well.

Reply:
As discussed in standup, we are planning on renaming the `ChannelManager` to `Wallet` and `Wallet` to `Engine`.